content/java/concepts/calendar/terms/clear.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: '.clear()'
+Description: 'A method designed to reset or clear specific fields of the calendar instance.'
+Subjects: 
+    - 'Computer Science'
```

Comment:
the spacing of the metadata section is incorrect. Please review one of our templates for reference-

https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md?plain=1

Reply:
sorry but i've not seen where my spacing is wrong in this.